data/sql/updates/pending_db_world/rev_1583667245426891200.sql

```diff
@@ -486,8 +487,8 @@ UPDATE `creature_template` SET `mindmg` = 624, `maxdmg` = 827, `DamageModifier`
 UPDATE `creature_template` SET `mindmg` = 484, `maxdmg` = 641, `DamageModifier` = 1 WHERE `entry` = 10505;
 -- Kirtonos the Herald
 UPDATE `creature_template` SET `mindmg` = 1161, `maxdmg` = 1538, `DamageModifier` = 1 WHERE `entry` = 10506;
--- The Ravenian (Not listed in Beastiary) Src: https://classicdb.ch/?npc=10507
-UPDATE `creature_template` SET `mindmg` = 832, `maxdmg` = 991 WHERE `entry` = 10507;
+-- The Ravenian (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=10507 / https://db.rising-gods.de/?npc=10507
+UPDATE `creature_template` SET `mindmg` = 362, `maxdmg` = 479, `DamageModifier` = 1 WHERE `entry` = 10507;
```

Comment:
here this one seems like he hits very lightly, especially for a bulky guy like him, don't you think?